arviz/plots/pairplot.py

```diff
@@ -85,10 +85,8 @@ def plot_pair(
     Examples
     --------
     KDE Pair Plot
-
     .. plot::
         :context: close-figs
-
```

Comment:
I think these black lines are important for docs to render properly, could you check all inline example images are shown?

Reply:
Yes, there was some problem with blank lines. I checked all examples are working fine now.

Reply:
Thanks!